### PR TITLE
[OSCD] regedit.exe LOLbas 72 [3]

### PR DESCRIPTION
**Pull request by SanWieb**
(second) Rebase of #1075

The rules cover:

General import (medium level)
ADS import (high level)
General export (low level)
Critical keys export (high level)
ADS export (high level)

### CODE REVIEW

File: rules/windows/sysmon/sysmon_regedit_export_to_ads.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+date: 2020/10/07
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection:
+        EventID: 15
+        Image|endswith: '\regedit.exe'
+    condition: selection
+fieds:
```
Comment (w0rk3r) [inline rules/windows/sysmon/sysmon_regedit_export_to_ads.yml:21]: Hey @SanWieb, there is a typo here, without it, the rules looks ready for me.

**Review by w0rk3r — CHANGES_REQUESTED** [review]

File: rules/windows/sysmon/sysmon_regedit_export_to_ads.yml (hunk 1 above)
Comment (SanWieb) [reply rules/windows/sysmon/sysmon_regedit_export_to_ads.yml]: Thanks a lot @w0rk3r.

The test is still failing, do you have an idea why?

Comment (w0rk3r) [reply rules/windows/sysmon/sysmon_regedit_export_to_ads.yml]: Every word in the title must start with a capital letter, that is the only reason that I can thin rn

Comment (SanWieb) [reply rules/windows/sysmon/sysmon_regedit_export_to_ads.yml]: Words like 'an' don't have to be a capital teller as far as I know. (https://capitalizemytitle.com/). Like this [4281cb20-2994-4580-aa63-c8b86d019934](https://github.com/Neo23x0/sigma/blob/master/rules/windows/process_creation/win_attrib_hiding_files.yml) rule for example.

But thank you for thinking along, maybe someone else knows what is going wrong here.

Comment (w0rk3r) [reply rules/windows/sysmon/sysmon_regedit_export_to_ads.yml]: Makes sense, looking at the logs, we can see that the test is breaking at the tags:

2020-10-15T13:42:57.0261255Z   File "/home/runner/work/sigma/sigma/tools/sigma/sigma2attack.py", line 32, in main
2020-10-15T13:42:57.0262161Z     for tag in tags:
2020-10-15T13:42:57.0263816Z TypeError: 'NoneType' object is not iterable

But there is nothing wrong with your rule, I'm going to see if I can figure out what's going on...i think you can mark this review as resolved.

**Comment by w0rk3r** [comment]
Found the error! https://github.com/Neo23x0/sigma/pull/1037#discussion_r505637014

**Comment by yugoslavskiy** [comment]
Hello @SanWieb! Well done! Your rules look perfect. Thank you very much!
Special thanks to @w0rk3r — you are doing a great job helping the participants!

**Review by yugoslavskiy — APPROVED** [review]
@thomaspatzke @Neo23x0 guys, I believe that this PR could be merged (:
please let us know what do you think about it if there is anything you would like to change.

**Pull request merged**